### DISCUSSION TITLE
ESP32 missing the last 3 pins on each side at the wifi antenna end

### DISCUSSION THREAD

**Discussion by capnrob97** [thehighnibble/thehighnibble.github.io#121] [IMSAI8080esp replica]
Is this normal? The reason I ask is bootup fails with unable to connect to external ram, the continuity tests on the surface mount all check out, but I am thinking it may be due to those missing pins on ESP32.

**Comment by pfcushing** [comment]
re-read step #1 of the step-by-instructions.  That will solve your concerns.

Paul Cushing

**Comment by capnrob97** [comment]
Yes, I found that right after I posted this. I was following along his step by step assembly video series, I assumed they covered everything but that didn’t come up in the videos. Sent from my iPadOn Nov 11, 2023, at 8:43 AM, pfcushing ***@***.***> wrote:﻿
re-read step #1 of the step-by-instructions.  That will solve your concerns.
Paul Cushing
